addons/mrp/views/mrp_production_views.xml

```diff
@@ -537,7 +537,7 @@
                     <field name="product_id"/>
                     <field name="product_variant_attributes"/>
                     <field name="move_raw_ids" string="Component" filter_domain="[('move_raw_ids.product_id', 'ilike', self)]"/>
-                    <field name="name" string="Work Center" filter_domain="[('bom_id.operation_ids.workcenter_id', 'ilike', self)]"/>
+                    <field name="workcenter_id" string="Work Center" filter_domain="[('bom_id.operation_ids.workcenter_id', 'ilike', self)]"/>
```

Comment:
hmm I didn't think about the domain itself actually 🤔, you might be right.

Reply:
I am not sure since we only redirect the user to this view when there are no work orders in the queue for this workcenter, in other words, when this workcenter could accept work order that are currently planned at other equivalent workcenters, and therefore perhaps allow the production to be done faster overall.